batch/batch/worker.py

```diff
@@ -34,7 +34,7 @@
 configure_logging()
 log = logging.getLogger('batch-worker')
 
-MAX_IDLE_TIME_MSECS = 5 * 60 * 1000
+MAX_IDLE_TIME_MSECS = 30 * 1000
```

Comment:
Wow! That seems aggressive. But I guess if the startup time is approximately 60-90 seconds then this makes sense?

Reply:
So basically watching batch load, if there's no work to do now, there won't be work to do for a while.

Here is an example: Konrad has big, slow jobs that gate many (~3K) medium fast jobs.  So when a big job finishes, we spin up 3K nodes and run jobs that take 5m.  Then we hang out for 5m more waiting for work (there is none), and then wind down, making the node usage about 50% efficient.

I actually want to make it smaller (5s? ... long enough that children of any jobs that just finished will get rescheduled) and add a minimum up time of 1m (since we pay for at least one minute of compute).  So I was planning to make it more aggressive in a later PR.

We need to start scraping the database and analyzing it.  We could actually quantify this explicitly.

Reply:
The startup seems closer to 1-2m.